packages/app-cli/app/command-keymaps.ts

```diff
@@ -0,0 +1,47 @@
+import BaseCommand from './base-command';
+import app from './app';
+import { _ } from '@joplin/lib/locale';
+const { cliUtils } = require('./cli-utils.js');
+
+class Command extends BaseCommand {
+	public override usage() {
+		return 'keymaps';
+	}
+
+	public override description() {
+		return _('Displays the configured keyboard shortcuts.');
+	}
+
+	public override compatibleUis() {
+		return ['cli', 'gui'];
+	}
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
+	public override async action(_args: any) {
+		const keymaps = await app().loadKeymaps();
+
+		this.stdout(_('Configured keyboard shortcuts:\n'));
+
+		const rows = [];
+		const padding = '  ';
+
+		rows.push([`${padding}KEYS`, 'TYPE', 'COMMAND']);
```

Comment:
```suggestion
		rows.push([`${padding}${_('KEYS')}`, _('TYPE'), _('COMMAND')]);
```

Consider using `_()` to make the header items localizable.

Reply:
Ok updated